src/accelerate/test_utils/testing.py

```diff
@@ -453,6 +471,13 @@ def require_torchdata_stateful_dataloader(test_case):
     )(test_case)
 
 
+def launches_subprocesses(test_case):
```

Comment:
this is to mark all tests that launch subprocesses, which should run first because once the current process uses/acquires the hpu device, subprocesses won't be able to acquire it.
we can also call it `should_run_first` as well.